zipkin-collector/rabbitmq/src/main/java/zipkin2/collector/rabbitmq/RabbitMQCollector.java

```diff
@@ -257,12 +257,13 @@ public void handleDelivery(String tag, Envelope envelope, BasicProperties props,
     for (int i = 0; i < addresses.size(); i++) {
       String[] splitAddress = addresses.get(i).split(":", 100);
       String host = splitAddress[0];
-      Integer port = null;
+      int port = -1;
       try {
         if (splitAddress.length == 2) port = Integer.parseInt(splitAddress[1]);
       } catch (NumberFormatException ignore) {
+        port = 0;
```

Comment:
Guess you might have added this for the same check but consider a comment instead